test/support/builders/atom_watcher_event.js

```diff
@@ -0,0 +1,70 @@
+const metadata = require('../../../core/metadata')
+const events = require('../../../core/local/steps/event')
+
+/*::
+import type { Stats } from 'fs'
+import type { AtomWatcherEvent, Action, Kind, Batch } from '../../../core/local/steps/event'
+*/
+
+function randomPick (elements /*: Array<T> */) /*: T */{
+  const l = elements.length
+  const i = Math.floor(Math.random() * Math.floor(l))
+  return elements[i]
+}
+
+module.exports = class AtomWatcherEventBuilder {
```

Comment:
I don't understand what's gained from writing all this code to fill in a shell of an object typed by Flow

Reply:
I'm not sure I fully understand your question but I wrote this builder to have default values for some properties I don't care about most of the time, have one-liners when creating test events and build the foundations for what could evolve if AtomWatcherEvent were to get more complex.

Reply:
Humm, I think you should care about the action for testing almost all the steps (but I'm not so sure about where this discussion is going ;-) )